Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/CapsuleLight.azsli

```diff
@@ -136,7 +136,7 @@ void ApplyCapsuleLight(ViewSrg::CapsuleLight light, Surface surface, inout Light
         float3 posToLight = closestIntersectionPoint - surface.position;
 
         // Tranmission contribution
-        lightingData.translucentBackLighting += GetBackLighting(surface, lightingData, lightIntensity, normalize(posToLight), 0.0);
+        lightingData.translucentBackLighting += GetBackLighting(surface, lightingData, lightIntensity, normalize(posToLight), -1.0f, 0.0f);
```

Comment:
nit - remark about the parameters here (-1.0 that gets ignored if I follow your code properly)

Reply:
New explanations have been added as context to the default parameters (https://github.com/o3de/o3de/pull/6428/commits/69df812941fedec065e23e1965fc646f66a50624)